manim/mobject/types/vectorized_mobject.py

```diff
@@ -2117,7 +2127,44 @@ def construct(self):
                         (gr-circle_red).animate.shift(RIGHT)
                     )
         """
-        return super().add(*vmobjects)
+
+        def get_type_error_message(invalid_obj, invalid_i):
+            return (
+                f"Only values of type {vmobject_render_type.__name__} can be added "
+                f"as submobjects of VGroup, but the value "
```

Comment:
(nitpck) not sure why linting didn't catch this, but you don't need it to be an f-string here?
```suggestion
                "as submobjects of VGroup, but the value "
```

Reply:
I implemented this